contracts/interfaces/ILightClient.sol

```diff
@@ -16,7 +16,14 @@
  */
 pragma solidity ^0.8.0;
 
-import {Ics23Proof, L1Header, OpL2StateProof} from "./IProofVerifier.sol";
+import {Ics23Proof} from "./IProofVerifier.sol";
+
+enum LightClientType {
+    DummyLightClient, // Note: not deployed on any mainnets
```

Comment:
nit: Should we use something like `SimTest` or similar instead of `Dummy`?

Reply:
done!